x-pack/plugins/maps/public/classes/layers/new_vector_layer_wizard/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export { NewVectorLayerEditor } from './wizard';
```

Comment:
The index file should export stuff used outside of the folder. NewVectorLayerEditor is not used outside of this folder but newVectorLayerWizardConfig is. How about exporting newVectorLayerWizardConfig instead?